include/tdep-ia64/libunwind_i.h

```diff
@@ -152,7 +152,7 @@ struct cursor
     unsigned int pi_is_dynamic :1; /* proc_info found via dynamic proc info? */
     unw_proc_info_t pi;         /* info about current procedure */
 
-    /* In case of stack-discontiguities, such as those introduced by
```

Comment:
This is not a spelling correction but a word choice change with subtle differences. I would keep the original word choice since it's more correct but remove the hyphen since hyphenated compound nouns are adjectives not objects.